Makefile

```diff
@@ -0,0 +1,8 @@
+PONY: help
+
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+format: ## Run linter and formatter 
+	dart fix --apply
+	dart format .
```

Comment:
Should there be some info in Read.me about it? How to run it or anything?

Reply:
Yes! Ive updated